plugins/inventory/virtualbox.py

```diff
@@ -56,15 +56,16 @@
 '''
 
 EXAMPLES = '''
+---
 # file must be named vbox.yaml or vbox.yml
-simple_config_file:
-    plugin: community.general.virtualbox
-    settings_password_file: /etc/virtulbox/secrets
-    query:
-      logged_in_users: /VirtualBox/GuestInfo/OS/LoggedInUsersList
-    compose:
-      ansible_connection: ('indows' in vbox_Guest_OS)|ternary('winrm', 'ssh')
+plugin: community.general.virtualbox
+settings_password_file: /etc/virtulbox/secrets
```

Comment:
```suggestion
settings_password_file: /etc/virtualbox/secrets
```

Reply:
Thanks for spotting that :) Will commit once #9552 is merged.